install/kubernetes/helm/istio-remote/templates/sidecar-injector-configmap.yaml

```diff
@@ -49,9 +49,13 @@ data:
         imagePullPolicy: {{ .Values.global.imagePullPolicy }}
         securityContext:
           capabilities:
+            drop:
+            - "all"
             add:
-            - NET_ADMIN
-          privileged: true
+            - "NET_ADMIN"
+            - "NET_RAW"
```

Comment:
I know `NET_ADMIN` is required. Why do we also need `NET_RAW`?

Reply:
`NET_ADMIN` is not enough, without `NET_RAW` injector init container unable to save iptables rules. 
Here is a simple test:
```
kubectl create ns inject-test

cat <<EOF | kubectl -n inject-test create -f -
apiVersion: v1
kind: Pod
metadata:
  name: istio-init-test
  labels:
    app: istio-init-test
spec:
  restartPolicy:
    Never
  containers:
  - name: istio-init-test
    image: gcr.io/istio-release/proxy_init:release-1.0-latest-daily
    securityContext:
      capabilities:
        drop:
        - "all"
        add:
        - "NET_ADMIN"
        - "NET_RAW"
      privileged: false
      allowPrivilegeEscalation: false
    args: ["-p", "15001", "-u", "1337", "-m", "REDIRECT", "-i", "*", "-x", "", "-b", "80", "-d", ""]
EOF

kubectl logs istio-init-test -n inject-test

```

Reply:
This seems to be platform specific, since it works for everybody else.

What platform are you using?

Reply:
CAP_NET_RAW
              * Use RAW and PACKET sockets;
              * bind to any address for transparent proxying.

The init container should not need this, and I cannot find enough documentation on this.

Reply:
I did more research, NET_RAW is enabled by default with docker container runtime, so we never needed to set it.

This PR drops all, so it needs to be enabled explicitly. Still not understanding why iptables need it...

Reply:
https://docs.docker.com/engine/reference/run/